settings.gradle.kts

```diff
@@ -112,7 +112,17 @@ gradle.beforeProject {
   group = "org.projectnessie"
 }
 
-include("code-coverage")
+listOf(
+    "code-coverage",
+    "nessie-deps-antlr",
+    "nessie-deps-build-only",
+    "nessie-deps-iceberg",
+    "nessie-deps-managed-only",
+    "nessie-deps-persist",
+    "nessie-deps-quarkus",
+    "nessie-deps-testing"
+  )
+  .forEach { include(it) }
```

Comment:
Just curious: is this syntax preferable for any reason? I guess multiple `include(...)` calls would take less lines :)

Reply:
Yea - it's just "less lines". spotless adds a blank line between each include :(

Reply:
TIL :)